packages/migrate/test/__snapshots__/migrate.test.ts.snap

```diff
@@ -1,10 +1,82 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`fix > .gts > component signature codefix > add component signature interface 1`] = `
+"import Component from \\"@glimmer/component\\";
+
+export interface WithMissingInterfaceSignature {
+  Args: {};
+}
+
+export default class WithMissingInterface extends Component<WithMissingInterfaceSignature> {
+  name = \\"Bob\\";
+
+  <template>
+    <span>Hello, I am {{this.name}} and I am {{@age}} years old!</span>
+    <span>My favorite snack is {{@snack}}.</span>
+  </template>
+}
+"
+`;
+
+exports[`fix > .gts > component signature codefix > adds \`Args\` property to component signature interface 1`] = `
+"import Component from \\"@glimmer/component\\";
+
+export interface WithMissingPropertySignature {
+  Args: {};
+}
+
+export default class WithMissingProperty extends Component<WithMissingPropertySignature> {
+  name = \\"Bob\\";
+
+  <template>
+    <span>Hello, I am {{this.name}} and I am {{@age}} years old!</span>
+    <span>My favorite snack is {{@snack}}.</span>
+  </template>
+}
+"
+`;
+
+exports[`fix > .gts > component signature codefix > adds missing args to Args to component signature interface 1`] = `
+"import Component from \\"@glimmer/component\\";
+
+export interface WithMissingArgsSignature {
+  Args: { age: any; snack: any };
```

Comment:
same as prior. why are we inferring `any` here (we should try and infer a type for `age` and `snack`